DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Package/WebApiLibraryPackage.cs

```diff
@@ -10,6 +14,10 @@ public void RegisterServices(Container container)
         {
             container.Register<IGithubService, GithubService>(Lifestyle.Singleton);
             container.Register<IAuthService, AuthService>(Lifestyle.Scoped);
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies().ToList();
+            container.Register(typeof (IValidator<>), assemblies, Lifestyle.Singleton);
+            container.RegisterConditional(typeof (IValidator<>), typeof (NullValidator<>), Lifestyle.Singleton,
```

Comment:
Something like that, the idea is that you should not iterate though the all assemblies in domain but only in those which contains implementation IValidator interface.